packages/docusaurus-theme-classic/src/theme/Navbar/Content/index.tsx

```diff
@@ -75,7 +76,7 @@ export default function NavbarContent(): JSX.Element {
           <NavbarItems items={rightItems} />
           <NavbarColorModeToggle className={styles.colorModeToggle} />
           {autoAddSearchBar && (
```

Comment:
No need for an extra variable now. Just `searchBarItem && (...)`

Reply:
Ok also my preferred way, wasn't quite sure if you want to preserve the naming there.